src/app/components/Copyright/index.tsx

```diff
@@ -0,0 +1,31 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { PropsWithChildren, useContext } from 'react';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import VisuallyHiddenText from '../VisuallyHiddenText';
+import Text from '../Text';
+import styles from './index.styles';
+
+const CopyrightContainer = ({ children }: PropsWithChildren) => {
+  const { dir, imageCopyrightOffscreenText, lang } = useContext(ServiceContext);
+
+  return (
+    <Text
+      as="p"
+      fontVariant="sansRegular"
+      size="minion"
+      css={[
+        styles.copyright,
+        { ...(dir === 'rtl' ? { right: 0 } : { left: 0 }) },
```

Comment:
Could we swap out dir here for [`inset-inline-start`](https://developer.mozilla.org/en-US/docs/Web/CSS/inset-inline-start)? Within the copyright styles?

Reply:
Ahh good idea! Think it should just be `inset-inline-start: 0;` right?

Reply:
I think so?

Reply:
Seems ok in the snapshots and Storybook.